content/en/platform/corda/5.0-beta/developing/api/application/flows.md

```diff
@@ -9,4 +9,4 @@ menu:
 section_menu: corda-5-beta
 ---
 
-The `flows` package contains interfaces and annotations for defining flows. The main interfaces are <a href="../../../../../../api-ref/corda/5.0-beta/kotlin/application/net.corda.v5.application.flows/-r-p-c-startable-flow/index.html" target="_blank">`RPCStartableFlow`</a> for flows expected to be started via the REST API, and <a href="../../../../../../api-ref/corda/5.0-beta/kotlin/application/net.corda.v5.application.flows/-responder-flow/index.html" target="_blank">`ResponderFlow`</a> for flows expected to be started via a peer-to-peer session. Annotations in this package are used to customize flow behaviour by marking properties for service injection (<a href="../../../../../../api-ref/corda/5.0-beta/kotlin/application/net.corda.v5.application.flows/-corda-inject/index.html" target="_blank">`@CordaInject`</a>) or marking either side of a peer-to-peer session (<a href="../../../../../../api-ref/corda/5.0-beta/kotlin/application/net.corda.v5.application.flows/-initiating-flow/index.html" target="_blank">`@InitiatingFlow`</a> and <a href="../../../../../../api-ref/corda/5.0-beta/kotlin/application/net.corda.v5.application.flows/-initiated-by/index.html" target="_blank">`@InitiatedBy`</a>).
\ No newline at end of file
+The `flows` package contains interfaces and annotations for defining flows. The main interfaces are <a href="../../../../../../api-ref/corda/5.0-beta/kotlin/application/net.corda.v5.application.flows/-r-p-c-startable-flow/index.html" target="_blank">`ClientStartableFlow`</a> for flows expected to be started via the REST API, and <a href="../../../../../../api-ref/corda/5.0-beta/kotlin/application/net.corda.v5.application.flows/-responder-flow/index.html" target="_blank">`ResponderFlow`</a> for flows expected to be started via a peer-to-peer session. Annotations in this package are used to customize flow behaviour by marking properties for service injection (<a href="../../../../../../api-ref/corda/5.0-beta/kotlin/application/net.corda.v5.application.flows/-corda-inject/index.html" target="_blank">`@CordaInject`</a>) or marking either side of a peer-to-peer session (<a href="../../../../../../api-ref/corda/5.0-beta/kotlin/application/net.corda.v5.application.flows/-initiating-flow/index.html" target="_blank">`@InitiatingFlow`</a> and <a href="../../../../../../api-ref/corda/5.0-beta/kotlin/application/net.corda.v5.application.flows/-initiated-by/index.html" target="_blank">`@InitiatedBy`</a>).
```

Comment:
I just notice the link here is "-r-p-c-startable-flow/index.html" wonder if we change this to client-startable-flow

Reply:
> I just notice the link here is "-r-p-c-startable-flow/index.html" wonder if we change this to client-startable-flow

These links are still pointing to the old KDocs. There is a separate ticket to update to the Javadcos when they're ready.